frontend/packages/dev-console/src/utils/pipeline-utils.ts

```diff
@@ -29,6 +31,8 @@ interface Resource {
   from?: string[];
 }
 
+const PIPELINE_SERVICE_ACCOUNT = 'pipeline';
```

Comment:
There is a `const.ts` at the top of the Pipelines folder, please move this to there.